src/ui/webbrowser/webbrowser.ts

```diff
@@ -1,44 +1,13 @@
-import Page from '../page';
+import NativeComponent from '../../core/native-component';
 import Color from '../color';
+import Page from '../page';
 
-export declare class AbstractWebBrowserOptions {
-  constructor(params?: Partial<AbstractWebBrowserOptions>);
-  get url(): string;
-  set url(value: string);
-  get barColor(): Color;
-  set barColor(value: Color);
-  /**
-   * Gets the itemColor.
-   *
-   * @returns {Color}
-   * @ios
-   * @since 2.0.11
-   */
-  get itemColor(): Color;
-  /**
-   * Sets the itemColor.
-   *
-   * @property {Color} [value = Color.BLACK]
-   * @ios
-   * @since 2.0.11
-   */
-  set itemColor(value: Color);
-  /**
-   * Gets the statusBar visibility.
-   *
-   * @returns {boolean}
-   * @ios
-   * @since 2.0.11
-   */
-  get statusBarVisible(): boolean;
-  /**
-   * Sets the statusBar visibility.
-   *
-   * @property {boolean} [value = true]
-   * @ios
-   * @since 2.0.11
-   */
-  set statusBarVisible(value: boolean);
+export interface IWebBrowserOptions extends NativeComponent {
+  url: string;
+  barColor?: Color;
+  ios?: Partial<{
```

Comment:
This shouldn't be used like this. You should create another interface for iOS&Android and use it to inherit `INativeMobileComponent`